verifier/src/lib.rs

```diff
@@ -107,7 +112,7 @@ impl<'a, V, O> VerificationTreeDisplay<'a, V, O> {
     }
 }
 
-impl<'a, V: VerificationMessage<O>, O: Clone> Display for VerificationTreeDisplay<'a, V, O> {
+impl<'a, V: VerificationMessage<O>, O> Display for VerificationTreeDisplay<'a, V, O> {
```

Comment:
This `Clone` was a stray requirement from eariler development it's not needed and was prohibiting the usage of `serde_json::Error` since that error doesn't support clone.